### PR TITLE
Sprint3/on 60

### PR DESCRIPTION
**Pull request by pinkcoderpro718**
1. Added main.py to do all the tasks as requested
2. Fixed all the errors with __init__.py
3. Cleaned the code as much as possible, with comments wherever necessary

### CODE REVIEW

**Review by 22jen — APPROVED** [review]
I find it working good.

File: news_sentiment_analysis/analyse_articles/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from openai import OpenAI
+
+client = OpenAI(api_key="YOUR_API_KEY")
```
Comment (Mohan-sharma-Venkatesh) [inline news_sentiment_analysis/analyse_articles/__init__.py:3]: move this lines 

```
from openai import OpenAI

client = OpenAI(api_key="YOUR_API_KEY")
```

to `news_sentiment_analysis/analyse_articles/sentimental_analysis.py`

File: news_sentiment_analysis/main.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# main.py
```
Comment (Mohan-sharma-Venkatesh) [inline news_sentiment_analysis/main.py:1]: remove comments

File: news_sentiment_analysis/main.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+from scrape_articles.get_article_urls.bing_news import GetArticlesURL
+from scrape_articles.get_article import GetArticles
+from analyse_articles.sentimental_analysis import analyze_sentiment_dataframe
+# from store_to_db.store_info import store_to_database
```
Comment (Mohan-sharma-Venkatesh) [inline news_sentiment_analysis/main.py:4]: remove comment

Comment (pinkcoderpro718) [reply news_sentiment_analysis/main.py:4]: That is the line of text used to store data to db, once the py file for that is written

Comment (Mohan-sharma-Venkatesh) [reply news_sentiment_analysis/main.py:4]: okay

**Pull request merged**